router/http/src/main/scala/io/buoyant/router/http/ApplyHostForwardedHeader.scala

```diff
@@ -0,0 +1,65 @@
+package io.buoyant.router.http
+
+import com.twitter.finagle._
+import com.twitter.finagle.http.{Fields, Request, Response, HeaderMap}
+import com.twitter.util.Future
+import java.net.{Inet4Address, Inet6Address, InetSocketAddress, SocketAddress, URI, URISyntaxException}
+import scala.collection.mutable
+import scala.util.Random
+import com.twitter.logging.Logger
+
+/**
+ * Applies the Host value in the [Forwarded](https://tools.ietf.org/html/rfc7239) header to the Request's Host header.
+ * So the Request is sent with the actual host value.
+ */
+class ApplyHostForwardedHeader() extends SimpleFilter[Request, Response] {
```

Comment:
Can we make this an object? (look at ViaHeaderAppenderFilter)

Reply:
Ok, done